docs/sources/articles/configuring.md

```diff
@@ -1,17 +1,37 @@
-page_title: Configuring Docker
-page_description: Configuring the Docker daemon on various distributions
-page_keywords: docker, daemon, configuration
+page_title: Configuring and running Docker
+page_description: Configuring and running the Docker daemon on various distributions
+page_keywords: docker, daemon, configuration, running, process managers
 
-# Configuring Docker on various distributions
+# Configuring and Running Docker on various distributions
 
 After successfully installing Docker, the `docker` daemon runs with it's default
-configuration. You can configure the `docker` daemon by passing configuration
-flags to it directly when you start it.  
+configuration.   
 
 In a production environment, system administrators typically configure the
 `docker` daemon to start and stop according to an organization's requirements.  In most
 cases, the system administrator configures a process manager such as `SysVinit`, `Upstart`,
-or `systemd` to manage the `docker` daemon's start and stop.
+, `systemd`, etc. to manage the `docker` daemon's start and stop.
```

Comment:
Not in favour of this change; ", etc." is superfluous here, because the list starts with "such as ..".

Please revert this change.